### ISSUE TITLE
[Feature] server packaging failed, maybe caused by document outdated

### ISSUE THREAD

**Issue by Thespica** [apache/incubator-hugegraph#2652]
### Feature Description (功能描述)

follow the [tutorial about server](https://hugegraph.apache.org/docs/quickstart/hugegraph-server/#33-source-code-compilation) but met error:

```bash
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   PerfUtilTest.testPerfUtilPerf:352 » CannotCompile by java.lang.reflect.Invocat...
[ERROR]   PerfUtilTest.testPerfUtilPerf4LightStopwatch:395 » CannotCompile by java.lang....
[ERROR]   PerfUtilTest.testPerfUtilWithProfilePackage:253 » CannotCompile by java.lang.r...
[ERROR]   RestClientTest.testPostHttpsWithAllParams:118 » Runtime java.net.SocketTimeout...
[ERROR]   RestClientTest.testPostHttpsWithTokenAndAllParams:135 » Runtime java.net.Socke...
[INFO]
[ERROR] Tests run: 342, Failures: 0, Errors: 5, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for hugegraph 1.5.0:
[INFO]
[INFO] hugegraph .......................................... SUCCESS [  1.161 s]
[INFO] hugegraph-commons .................................. SUCCESS [  0.685 s]
[INFO] hugegraph-common ................................... FAILURE [ 35.086 s]
[INFO] hugegraph-pd ....................................... SKIPPED
[INFO] hg-pd-grpc ......................................... SKIPPED
[INFO] hg-pd-common ....................................... SKIPPED
[INFO] hg-pd-client ....................................... SKIPPED
[INFO] hugegraph-store .................................... SKIPPED
[INFO] hg-store-common .................................... SKIPPED
[INFO] hugegraph-server ................................... SKIPPED
[INFO] hugegraph-core ..................................... SKIPPED
[INFO] hugegraph-rpc ...................................... SKIPPED
[INFO] hugegraph-api ...................................... SKIPPED
[INFO] hugegraph-cassandra ................................ SKIPPED
[INFO] hugegraph-scylladb ................................. SKIPPED
[INFO] hugegraph-rocksdb .................................. SKIPPED
[INFO] hugegraph-mysql .................................... SKIPPED
[INFO] hugegraph-palo ..................................... SKIPPED
[INFO] hugegraph-hbase .................................... SKIPPED
[INFO] hugegraph-postgresql ............................... SKIPPED
[INFO] hg-store-grpc ...................................... SKIPPED
[INFO] hg-store-client .................................... SKIPPED
[INFO] hugegraph-hstore ................................... SKIPPED
[INFO] hugegraph-dist ..................................... SKIPPED
[INFO] hugegraph-example .................................. SKIPPED
[INFO] hugegraph-test ..................................... SKIPPED
[INFO] hg-pd-core ......................................... SKIPPED
[INFO] hg-pd-service ...................................... SKIPPED
[INFO] hg-pd-dist ......................................... SKIPPED
[INFO] hg-pd-test ......................................... SKIPPED
[INFO] hg-store-rocksdb ................................... SKIPPED
[INFO] hg-store-core ...................................... SKIPPED
[INFO] hg-store-node ...................................... SKIPPED
[INFO] hg-store-test ...................................... SKIPPED
[INFO] hg-store-dist ...................................... SKIPPED
[INFO] hg-store-cli ....................................... SKIPPED
[INFO] install-dist ....................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  37.459 s
[INFO] Finished at: 2024-08-31T17:47:19+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.20:test (default-test) on project hugegraph-common: There are test failures.
[ERROR]
[ERROR] Please refer to /home/john/hugegraph/tmp/hugegraph/hugegraph-commons/hugegraph-common/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :hugegraph-common
```

I guess the directory should be `*hugegraph/*server`, rather than `*hugegraph`?

PS: maybe this issue should be marked as question, and be posted in documentation repository?

**Comment by VGalaxies** [comment]
We have noticed this issue and fixed it in #2651...

**Comment by Thespica** [comment]
Thank you!
I have ignored that.
